french-soup/index.html

```diff
@@ -5,17 +5,51 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Nice work, however, you've placed this solution in the wrong folder. It should be in ```apple-pie``` and not ```french-soup```.